sdk/README.md

```diff
@@ -113,19 +113,34 @@ in the step above, we need to deploy the generated Tekton YAML to our Kubernetes
 cluster with `kubectl`. The Tekton server will automatically start a pipeline run
 for which we can follow the logs using the `tkn` CLI:
 
-    kubectl apply -f pipeline.yaml
+    kubectl apply -f pipeline.yaml -n kubeflow
```

Comment:
we need to add a sentence above this code snippet explaining about deploying the pipeline to the `kubeflow` namespace

Reply:
why is it tied to kubeflow namespace @Tomcli ?

Reply:
I updated the readme in my latest commit. It's because the minio secret is only available inside the Kubeflow namespace for the latest Kubeflow release.  https://github.com/kubeflow/manifests/blob/master/pipeline/minio/base/secret.yaml

Reply:
This is for artifact tracking